R/infection_process.R

```diff
@@ -0,0 +1,53 @@
+#' @title Calculating the FOI
```

Comment:
Looks like this is introduced in #33 so I presume this PR is queued behind that?